release-notes.txt

```diff
@@ -1,3 +1,366 @@
+RIOT-2019.01 - Release Notes
+============================
+RIOT is a multi-threading operating system which enables soft real-time
+capabilities and comes with support for a range of devices that are typically
+found in the Internet of Things: 8-bit and 16-bit microcontrollers as well as
+light-weight 32-bit processors.
+
+RIOT is based on the following design principles: energy-efficiency, soft
+real-time capabilities, small memory footprint, modularity, and uniform API
+access, independent of the underlying hardware (with partial POSIX compliance).
+
+RIOT is developed by an international open-source community which is
+independent of specific vendors (e.g. similarly to the Linux community) and is
+licensed with a non-viral copyleft license (LGPLv2.1), which allows indirect
+business models around the free open-source software platform provided by RIOT.
+
+
+About this release:
+===================
+
+The 2019.01 release includes a number of new features including DMA support on
```

Comment:
Sorry for my late review.

I think the IPv6 refactor should be also included. The [project](https://github.com/RIOT-OS/RIOT/projects/24) was finished during January, just before feature freeze

Reply:
Ok, it's not highlighted but it's added in the list of changes [here](https://github.com/RIOT-OS/RIOT/pull/10871/files#diff-acce5952fb2ed7b16765d3ddab1e2403R53). I think it's fine even if not perfect.

Reply:
yes, sure ;)